components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Client Runs"
+
+date = 2021-03-12T12:01:58-07:00
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Chef Infra Server"
+    parent = "automate"
+    identifier = "automate/chef_infra_server.md Chef Infra Server"
+    weight = 60
++++
+
+## Overview
+
+The __Chef Server__ page shows all the Chef Infra Servers connected to Chef Automate. The __Chef Server__ user interface can be used to view and manage all the Chef Infra Server Objects like:
+
+- Cookbooks
+- Roles
+- Environments
+- Data Bags
+- Clients
+
+## Connect to the Chef Infra Server
+
+To start making use out of the Chef Infra Server, firstly deploy and run the [Chef Automate](https://docs.chef.io/automate/install/#installation-guide).
+
+**Note:** No explicit configuration is required to install the Automate Infra views.
+
+Login to Chef Automate using the credentials. Once logged in, the page looks like as shown below:
+
+![Chef Automate](/images/automate/chef-automate-on-chef-infra-page.png)
+
+Click on **Infrastructure** located at the top bar of the page. Refer to the image below:
+
+![Chef Automate with Highlighted Infrastructure](/images/automate/chef-automate-infrastructure.png)
```

Comment:
Also another image of the Event feed?

Ok, I think I get what you're trying to do. It is OK to say "To view your Chef Infra Servers in Automate, select **Infrastructure > Chef Infra Server**" or something similar. We can assume that the have an Automate instance running and know how to navigate tabs in a UI.